hooks/post_gen_project.py

```diff
@@ -33,6 +42,18 @@ def http2ssh(url):
 
 
 def post_gen():
+    {% if cookiecutter.documentation_engine != 'Sphinx' %}
```

Comment:
took me hours to figure out why Jinja2 wasn't recognizing elif states in this case. There were no traceback errors